certified-connectors/Nextcom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,597 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nextcom",
+    "description": "Connect your application data flow to Nextcom to optimize sales and customer management.",
+    "version": "1.0",
+	"contact": {
+      "name": "Nextcom Support",
+      "url": "https://nextcom.no/en/services/support/",
+      "email": "post@nextcom.no"
+    }
+  },
+  "host": "dev.nextcom.no",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+  "application/json"
+],
+  "produces": [],
+  "paths":{
+    "/crm-system/contacts": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Returns the new contact ID, or an existing contact ID if a duplicate was hit",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ContactId",
+                  "title": "ContactId"
+                },
+                "insertStatus": {
+                  "type": "string",
+                  "description": "Relays the contact ID and, wether a new contact was created or if a duplicate was found and re-used.",
+                  "title": "Insert status"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Returns error if the input model is malformed",
+            "schema": {
+				"title": "Bad request error",
```

Comment:
@ebarland Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Fixed: https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/dd2b68272bc676d9fe8d785009643b31e36a5b12